bundler/spec/commands/fund_spec.rb

```diff
@@ -55,6 +55,42 @@
     expect(out).to_not include("gem_with_dependent_funding")
   end
 
+  it "does not consider fund information for uninstalled optional dependencies" do
+    install_gemfile <<-G
+      source "#{file_uri_for(gem_repo2)}"
+      group :whatever, optional: true do
+        gem 'has_funding_and_other_metadata'
+      end
+      gem 'has_funding'
+      gem 'rack-obama'
+    G
+
+    bundle "fund"
+
+    expect(out).to include("* has_funding (1.2.3)\n  Funding: https://example.com/has_funding/funding")
+    expect(out).to_not include("has_funding_and_other_metadata")
+    expect(out).to_not include("rack-obama")
+  end
+
+  it "considers fund information for installed optional dependencies" do
+    install_gemfile <<-G
+      source "#{file_uri_for(gem_repo2)}"
+      group :whatever, optional: true do
+        gem 'has_funding_and_other_metadata'
+      end
+      gem 'has_funding'
+      gem 'rack-obama'
+    G
+
+    bundle "config set with whatever"
```

Comment:
Sorry, I meant _before_ `install_gemfile` above, so that you can remove the line below and don't need to run `bundle "install"` again.

Reply:
Ah, right. Yeah, makes sense. It didn't really register that `install_gemfile` does that already

Reply:
No problem, it was something very minor! :)